client/src/router/index.js

```diff
@@ -176,7 +185,7 @@ router.beforeEach(async (to, from, next) => {
     next("/login");
   } else if (to.path == "/" && token) {
     await userService.refresh_token();
-    next("/home");
+    next("/home")
```

Comment:
why removing the semicolon?

Reply:
> why removing the semicolon?

it was meant to do other changes, but I commented then deleted them and forgot to retype the ; 
will fix it